contribute-to-our-docs.rst

```diff
@@ -0,0 +1,39 @@
+Contribute to our docs
+======================
+
+We are committed to continuously improving our documentation and making it more useful for both ourselves and others.
+
+**We warmly welcome community contributions, suggestions, fixes, and constructive feedback.**
+
+If you feel something is unclear, wrong, or broken, please don't hesitate to reach out to us by any of these means:
+
+- Join the `#launchpad IRC channel`_ at `irc.libera.chat`_ and ask your
+  questions.
+- Ask a question: https://answers.launchpad.net/launchpad
+- Send an e-mail to feedback@launchpad.net
+
+Or go ahead and submit an issue or pull request at https://github.com/canonical/launchpad-manual.
+
+Feel free to suggest changes to any documentation topic with updated or more insightful information. The navigation structure, style, and content of our documentation follows the `Diátaxis`_ systematic framework for technical documentation authoring. This splits pages into tutorials, how-to guides, reference material, and explanatory text.
+
+Style and language
+++++++++++++++++++
+
+Launchpad's documentation is aimed at helping both beginners and experts. As such, we try to use appropriate writing for the subject, to use inclusive language, and to assume that the reader has as little prior knowledge as possible.
+
+Some general tips:
+
+- use a spell checker
```

Comment:
We have a linter for that, so at one point we probably should suggest to use tooling, though I do not want anybody to be kept off from contributing when they do not feel comfortable to use make files.